bedrock/security/templates/security/client-bug-bounty.html

```diff
@@ -168,7 +168,7 @@ <h2 id="exploit-mitigation-bounty">Exploit Mitigation Bug Bounty</h2>
 
 <h2 id="static-analysis-bounty">Static Analysis Bounty</h2>
 
-  <p>In coordination with the GitHub Security Lab, we have launched a new program that rewards the submission of static analysis tools that identify present or historical security vulnerabilities in Firefox.  We will accept static analysis queries written in CodeQL or as clang-based checkers (clang analyzer, clang plugin using the AST API or clang-tidy). Submissions should be made following <a href="#claiming-a-bounty">our instructions below</a>.</p>
+  <p>We also have a program that rewards the submission of static analysis tools that identify present or historical security vulnerabilities in Firefox.  We will accept static analysis queries [written as clang-based checkers](https://firefox-source-docs.mozilla.org/code-quality/static-analysis/writing-new/index.html) - we have some documentation that [may help you get started or integrate and run over Firefox as a whole](https://firefox-source-docs.mozilla.org/code-quality/static-analysis/writing-new/index.html). Submissions should be made following <a href="#claiming-a-bounty">our instructions below</a>.</p>
```

Comment:
Change Markdown link to HTML anchor

```suggestion
  <p>We also have a program that rewards the submission of static analysis tools that identify present or historical security vulnerabilities in Firefox.  We will accept static analysis queries <a href="https://firefox-source-docs.mozilla.org/code-quality/static-analysis/writing-new/index.html">written as clang-based checkers</a> - we have some documentation that <a href="https://firefox-source-docs.mozilla.org/code-quality/static-analysis/writing-new/index.html">may help you get started or integrate and run over Firefox as a whole</a>. Submissions should be made following <a href="#claiming-a-bounty">our instructions below</a>.</p>
```